src/ladder/OutputView.java

```diff
@@ -0,0 +1,34 @@
+package ladder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OutputView {
+
+    private List<String> stringElements;
```

Comment:
```suggestion
    private List<char> stringElements;
```
아니면 그냥 `char[] elements = {'|', ... } ` 해도 되지 않을까요?